org.eclipse.jdt.text.tests/src/org/eclipse/jdt/text/tests/codemining/ParameterNamesCodeMiningTest.java

```diff
@@ -77,7 +78,12 @@ public void setUp() throws Exception {
 			closeIntro(PlatformUI.getWorkbench());
 		}
 		fProject= JavaProjectHelper.createJavaProject(getClass().getName(), "bin");
-		JavaProjectHelper.addRTJar(fProject);
+		JavaProjectHelper.addRTJar_16(fProject, true);
+
+		Map<String, String> options= fProject.getOptions(false);
+		JavaProjectHelper.set17_CompilerOptions(options);
```

Comment:
rt.jar from 16, but compiler options for 17?

Reply:
Looks like `addRTJar_17(..)` didn't exist so I probably just took the path of least resistance in the tests. I'll go ahead and create the method.